pkg/controller/volume/pvcprotection/pvc_protection_controller.go

```diff
@@ -162,33 +192,34 @@ func (c *Controller) processPVC(pvcNamespace, pvcName string) error {
 	pvc, err := c.pvcLister.PersistentVolumeClaims(pvcNamespace).Get(pvcName)
 	if apierrors.IsNotFound(err) {
 		klog.V(4).Infof("PVC %s/%s not found, ignoring", pvcNamespace, pvcName)
-		return nil
+		return false, nil
 	}
 	if err != nil {
-		return err
+		return false, err
```

Comment:
maybe return true, err since in this case pvc will be requeued?

Reply:
Yes we can retry here since its not a not found error.